Doesn't compile with resolver="2" on Windows

Steps to reproduce:
1. Create a new project with the following `Cargo.toml`:
    ```
    [package]
    name = "example"
    version = "0.1.0"
    edition = "2018"
    resolver = "2"

    [dependencies]
    howlong = "0.1"
    ```
2. Run `cargo check`

I get the following error:

```
error[E0432]: unresolved imports `winapi::um::errhandlingapi`, `winapi::um::processthreadsapi`
 --> C:\Users\username\.cargo\registry\src\github.com-1ecc6299db9ec823\howlong-0.1.6\src\clock\win.rs:6:5
  |
6 |     errhandlingapi::GetLastError,
  |     ^^^^^^^^^^^^^^ could not find `errhandlingapi` in `um`
7 |     processthreadsapi::{GetCurrentProcess, GetCurrentThread, GetProcessTimes, GetThreadTimes},
  |     ^^^^^^^^^^^^^^^^^ could not find `processthreadsapi` in `um`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,12 @@ errno = "0.2"
 libc = "0.2"
 
 [target.'cfg(windows)'.dependencies]
-winapi = { version = "0.3", features = ["profileapi", "sysinfoapi"] }
+winapi = { version = "0.3", features = [
+    "profileapi",
+    "sysinfoapi",
+    "errhandlingapi",
+    "processthreadsapi",
+] }
 
 [dev-dependencies]
 criterion = "0.3"

